src/metadata.jl

```diff
@@ -28,6 +33,38 @@ function _pkgs()
     return _CACHE[:PKGS]
 end
 
+function _env()
+    if isempty(_CACHE[:PROJECT]) && isempty(_CACHE[:MANIFEST])
+        _CACHE[:PROJECT] = read(Base.active_project(), String)
+        _CACHE[:MANIFEST] = read(
+            joinpath(dirname(Base.active_project()), "Manifest.toml"),
+            String
+        )
+    end
+
+    return (_CACHE[:PROJECT], _CACHE[:MANIFEST])
+end
+
+# Used to convert the old "PkgName" => VersionNumber metadata to a
+# Project.toml and Manifest.toml file.
+function _env(pkgs::Dict)
```

Comment:
I think we should make a `upgrade_jlso.jl`  file and put all these together

Reply:
Hmmm, but we also call the no-args version in the standard JLSOFIle constructor. It seems weird to spread those definitions across multiple files.

Reply:
rename this one to `_upgrade_env` or something?

Reply:
Renamed and moved to a deprecated file.